R/file_utils.R

```diff
@@ -536,7 +536,8 @@ load_regional_multivariate_data <- function(matrix_y_min_complete = NULL, # when
     X = X,
     maf = apply(X, 2, compute_maf),
     chrom = dat$chrom,
-    grange = dat$grange
+    grange = dat$grange,
+    X_variance= matrixStats::colVars(X)
```

Comment:
@Chunmingl please use @importFrom matrixStats colVars in your roxygen strings. Dont use `::` in the code. This is an R package you need to follow the rules.

Reply:
This is addressed in the new commit.